app/pages/lab/feedback/types/common-form-hoc.jsx

```diff
@@ -0,0 +1,152 @@
+counterpart.registerTranslations('en', {
+  feedbackEditForm: {
+    title: 'Edit Feedback',
+    fields: {
+      id: {
+        title: 'ID',
+        help: 'A unique field used to identify a feedback type.'
+      },
+      successEnabled: {
+        title: 'Enable positive feedback',
+        help: 'Shows a feedback message to the user when they correctly identify a known subject.'
+      },
+      defaultSuccessMessage: {
+        title: 'Default success message',
+        help: 'The message to show to the volunteer when they make a correct classification. Can be overridden using subject metadata.'
+      },
+      failureEnabled: {
+        title: 'Enable negative feedback',
+        help: 'Shows a feedback message to the user when they incorrectly identify a known subject.'
+      },
+      defaultFailureMessage: {
+        title: 'Default failure message',
+        help: 'The message to show to the volunteer when they make an incorrect classification. Can be overridden using subject metadata.'
+      }
+    }
+  }
+});
+/* eslint-enable max-len */
+
+function returnCommonFormHOC(FeedbackTypeForm) {
+  class CommonFormHOC extends Component {
+    constructor(props) {
+      super(props);
+      this.handleInputChange = this.handleInputChange.bind(this);
+      this.handleSave = this.handleSave.bind(this);
+      this.handleUpdateState = this.handleUpdateState.bind(this);
+      this.renderCommonFields = this.renderCommonFields.bind(this);
+      this.validate = this.validate.bind(this);
+
+      const { feedback } = props;
+      this.state = {
+        form: {
+          id: feedback.id || '',
+          successEnabled: (typeof feedback.successEnabled === 'boolean') ? feedback.successEnabled : true,
+          defaultSuccessMessage: feedback.defaultSuccessMessage || '',
+          failureEnabled: (typeof feedback.failureEnabled === 'boolean') ? feedback.failureEnabled : true,
+          defaultFailureMessage: feedback.defaultFailureMessage || ''
+        },
+        validations: [
+          // Must have an id
+          form => (!!form.id),
+          // Must have either success or failure or both enabled
+          form => (!(!form.successEnabled && !form.failureEnabled)),
+          // Success must be disabled, or enabled with a message
+          form => (!form.successEnabled || (form.successEnabled &&
+            form.defaultSuccessMessage && form.defaultSuccessMessage.length > 0)),
+          // Failure must be disabled, or enabled with a message
+          form => (!form.failureEnabled || (form.failureEnabled &&
+            form.defaultFailureMessage && form.defaultFailureMessage.length > 0))
+        ]
+      };
+    }
+
+    handleInputChange({ target }) {
+      this.handleUpdateState({
+        form: {
+          [target.name]: (target.type === 'checkbox') ? target.checked : target.value
+        }
+      });
+    }
+
+    handleSave() {
+      this.props.onSubmit(this.state.form, this.props.index)
```

Comment:
@srallen do you know how to trigger this? I can't reproduce with neither feedback enabled nor disabled.